third_party/move/move-compiler-v2/tests/file-format-generator/index_acquires_err.exp

```diff
@@ -0,0 +1,28 @@
+
+Diagnostics:
+error: Invalid operation: storage operation on type `test::Y` can only be done within the defining module `0x42::test`
+   ┌─ tests/file-format-generator/index_acquires_err.move:14:9
+   │
+14 │     fun test_resource_other_module() acquires 0x42::test::Y {
+   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^
+15 │         let addr = @0x1;
+16 │         assert!((&0x42::test::Y<0x42::test::X<bool>>[addr]).field.value == true, 1);
+   │                 ------------------------------------------- called here
```

Comment:
It would be useful to be specific about the operation here.  `global resource test::Y<..> at addr borrowed here`.  Or  even `borrow_global<0x42::test::Y<0x42::test::X<bool>>(addr)` called here.  With the use of `[]` it's really unclear what could be "called" here.